### DISCUSSION TITLE
镜像源 经常超时，怎么换个 国内的

### DISCUSSION THREAD

**Discussion by abc714409484** [httprunner/httprunner#1515] [2 - 答疑（Q&A）]
![image](https://user-images.githubusercontent.com/38741562/198878047-843ee90c-dfe4-426c-93a1-8801da374c35.png)
![image](https://user-images.githubusercontent.com/38741562/198878064-0a5f539e-cd6d-4681-88a1-c6d83a1b28f2.png)

**Comment by abc714409484** [comment]
怎么切换 默认的 镜像 域名

**Reply by dongli0101** [reply]
顶一个，太特么坑了

**Comment by NQ31** [comment]
你指定自己python虚拟环境路径就可以了
